docs/glossary.md

```diff
@@ -0,0 +1,97 @@
+
+## Precedence
+
+The principle of precedence is used to determine which liturgical celebration should be observed when two or more celebrations fall on the same day. The principle of precedence is based on the liturgical rank of the celebrations.
+
+See GNLYC §59.
+
+## Proper of Saints (or Sanctoral)
+
+The "Proper of Saints" (or "Sanctoral") consists of the fixed feasts, celebrated on the very same date each year (no matter what the day of the week),including Christmas and all the saints' days.
+
+## Proper of Time (or Temporal)
+
+The "Proper of Time" (or "Temporal") consists of the movable feasts, most of them keyed to Easter (which falls on a different Sunday every year), including Ascension, Pentecost, and so on.
+
+Note: we avoid using the term "Temporal" to prevent potential confusion with the `Temporal` object in JavaScript, which is still in a [proposal stage](https://tc39.es/proposal-temporal/docs/).
+
+## Psalter Week Cycle
+
+The four-week cycle of the psalter is coordinated with the liturgical year in such a way that on the First Sunday of Advent, the First Sunday in Ordinary Time, the First Sunday of Lent, and Easter Sunday the cycle is always begun again with Week 1 (others being omitted when necessary).
+
+See GILH §133.
+
+## Rank
+
+The rank of a liturgical celebration is a measure of its importance in the liturgical calendar. The rank of a celebration determines how it should be observed when it falls on the same day as another celebration.
+
+There are five categories of ranks: solemnities, Sunday, feasts, memorials, weekday. A memorial can be obligatory or optional.
+
+See GNLYC §3-16.
+
+## Sanctoral
+
+See [Proper of Saints]().
+
+## Season
+
+Liturgical seasons are the periods of time in the liturgical year that are marked by a particular liturgical color and have a particular focus. The liturgical seasons are Advent, Christmastide, Lent, Easter Time, and Ordinary Time. The Ordinary Time is divided into two parts: the first part begins on the Monday after the Baptism of the Lord and ends on the day before Ash Wednesday; the second part begins on the Monday after Pentecost and ends on the Saturday before the First Sunday of Advent.
+
+## Sunday Cycles
+
+A three-year cycle for Sunday Mass readings (and some solemnities), designated by A, B, or C. Each cycle begins on the First Sunday of Advent of the previous civil year and ends on Saturday after the Christ the King Solemnity. The cycles follow each other in alphabetical order. C year is always divisible by 3, A has remainder of 1, and B remainder of 2.
+
+## Temporal
+
+See [Proper of Time]().
+
+## Weekday Cycles
```

Comment:
I would use the term on singular, unless it has a unique/special meaning in plural.

```suggestion
## Weekday Cycle
```